.github/workflows/vacuum.yml

```diff
@@ -0,0 +1,23 @@
+name: Vacuum OpenAPI lint
+on:
+  pull_request:
+    paths:
+      - 'zeebe/gateway-protocol/src/main/proto/rest-api.yaml'
+
+jobs:
+  lint-openapi-spec:
+    runs-on: ubuntu-latest
+    steps:
+      - id: depth
+        name: Determine fetch depth for checkout
+        run: echo "depth=$(expr ${{ github.event.pull_request.commits }} + 1)" >> $GITHUB_OUTPUT
```

Comment:
❓ This question is very unimportant to this PR, but can you help me understand why the depth is not consistent across PRs?

Reply:
Can you detail a bit what you mean with
> not consistent across PRs

This depth calculation is required because we want to investigate **all** commits of a PR. Otherwise, by default, we only fetch the latest commit for every PR which isn't enough for this workflow but suffices for most other workflows.

Hope that touches on what you were wondering about 😅

Reply:
Good point, we actually don't need this here. This ensures we fetch the commit history for a PR and not just the latest commit (default of the checkout action). We should be fine with the latest commit since we're interested in the content, not the commit history 👍